.circleci/create_circleci_config.py

```diff
@@ -401,6 +409,36 @@ def job_name(self):
     tests_to_run="tests/repo_utils",
 )
 
+doc_test_job = CircleCIJob(
+    "pr_documentation_tests",
+    additional_env={"TRANSFORMERS_VERBOSITY":"error", "DATASETS_VERBOSITY":"error", "SKIP_CUDA_DOCTEST": "1"},
```

Comment:
Just for my own understanding and doulbe checking - this means cuda docstests are skipped for the doc test job?

Reply:
On circleCI only, which does not have GPU runners.